frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature

```diff
@@ -0,0 +1,62 @@
+@add-flow
+Feature: Create Application using Templates
+              As a user, I want to deploy the application using Templates provided in Developer Catalog
+
+        Background:
+            Given user is at developer perspective
+              And user is at Developer Catalog page
+
+
+        Scenario: Deploy CI/CD Application using Catalog Templates
+            Given user has created or selected namespace "aut-addflow-templates"
+             When user selects "Templates" option from Type section
+              And user selects CI/CD type
```

Comment:
Same given statement for all scenarios, can we move it to Background?
```suggestion
            When user selects CI/CD type
```

Reply:
Updated